### ISSUE TITLE
Auto silence ringer

### ISSUE THREAD

**Issue by eevleevs** [Freeyourgadget/Gadgetbridge#742]
I would like to suggest an option to automatically silence the phone ringer when the wearable is connected.

**Comment by IzzySoft** [comment]
Good idea. I don't know why, but my phone recently started doing that by itself for notifications. But it might just have lost some setting :wink:

**Comment by gianlucapir** [comment]
Really a good idea, it would be nice if you could implement it

**Comment by cpfeiffer** [comment]
IMHO we should interface with LibreTasks or Easer for automating things.

**Comment by IzzySoft** [comment]
@cpfeiffer Meanwhile, are there any intents one could use to have Tasker perform actions – like having GB send a notification with a given text to the connected wearable, or switch a given setting?
